descriptors/ModuleDescriptor-template.json

```diff
@@ -66,7 +66,7 @@
     },
     {
       "id": "organizations-storage.contacts",
-      "version": "1.1",
+      "version": "2.0",
```

Comment:
Why it should be bumped up to 2.0? I can't find any breaking changes for contacts

Reply:
`contact.json` has a reference to `phone_number.json` schema which was significantly changed. So, contacts interface no more compatible with version 1.x

Reply:
Didn't notice, thank you for pointing it out.